src/App.tsx

```diff
@@ -1,24 +1,58 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useContext, useEffect, useMemo } from 'react';
 
-const USER_ID = 0;
+import { Header } from './components/Header';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Notifications } from './components/Errors';
+import { getTodos } from './api/todos';
+import { filterTodo } from './helpers/filterTodo';
+import { UserWarning } from './UserWarning';
+import { ErrorType } from './types/Errors';
+import { USER_ID } from './utils/userId';
+import { AppContext } from './AppContext';
 
 export const App: React.FC = () => {
+  const {
+    todos,
+    setTodos,
+    filterBy,
+    shouError,
+  } = useContext(AppContext);
```

Comment:
You can create a custom hook to avoid passing context in useContext function every time
```typescript
const useAppContext = () => useContext(AppContext);
```